traffic_ops/bin/traffic_ops_ort.pl

```diff
@@ -783,6 +771,32 @@ sub get_print_current_client_connections {
 	( $log_level >> $DEBUG ) && print "DEBUG There are currently $current_connections connections.\n";
 }
 
+sub get_update_status {
+	my $uri     = "/api/1.3/servers/$hostname_short/update_status";
```

Comment:
any reason you're not passing in `$hostname_short`?  best not to rely on global vars..

Reply:
The entire ORT script relies on globals.  $hostname_short is set at the start of the run and isn't really passed at all.  I agree with you, but we're talking about rewrite-level changes for that, aren't we?

Reply:
in this case, the global is available at each point `get_update_status` is called.  So you can (and should) pass in to this sub,  but not restructure the rest of the code (agree there....)   Just makes it explicit that that's what's being used there.

Reply:
so just 

my ($upd_json, $uri) = get_update_status($hostname_short);

for example